menus/templatetags/menu_tags.py

```diff
@@ -157,16 +157,14 @@ def get_context(self, context, from_level, to_level, extra_inactive,
             children = cut_levels(nodes, from_level, to_level, extra_inactive, extra_active)
             children = menu_renderer.apply_modifiers(children, namespace, root_id, post_cut=True)
 
-        try:
-            context['children'] = children
-            context['template'] = template
-            context['from_level'] = from_level
-            context['to_level'] = to_level
-            context['extra_inactive'] = extra_inactive
-            context['extra_active'] = extra_active
-            context['namespace'] = namespace
-        except:
-            context = {"template": template}
+        context['children'] = children
```

Comment:
I don't think this should be changed, it was clearly catching something in the past to have a try except block. Do you know what it was catching before and why it's no longer required?

Reply:
I couldn't see how it could be catching anything as its not really doing anything.

Reply:
@Aiky30 I think it was written in 2010 to catch an error on `children`, but since then theres a default of `children = []`
This is where it started;
https://github.com/django-cms/django-cms/commit/bda49b26cd5f1216dfadb756ec3b693348295d3f#diff-2f98e9f55ac912e040227934eccb938c8a4d4b511f0f3222466753e17a23f713R80

Reply:
Yup agreed, that looks really bad. Haha